### DISCUSSION TITLE
Is there a suggested method to promote awareness for drag and drop abilities on mobile?

### DISCUSSION THREAD

**Discussion by Stanman519** [hello-pangea/dnd#727] [Q&A] [answered]
On pc, you get the hand cursor to show you can drag and drop. Is there anything built-in to help users realize they can drag and drop components when on mobile? 

I was thinking of adding a occasional pulsing bounce on one of the components, but I was curious if there was something else I should be aware of.

Thanks! Using this library has been awesome.

**Comment by 100terres** [comment, marked as answer]
I think the user experience you are suggesting could be interesting to let users know that something is draggable, but it is out of the scope of this library. You can always implement an onboarding tool to let your users know how to use your application. Maybe something like [driver.js](https://driverjs.com/) could help you.
